xmake/modules/package/manager/install_package.lua

```diff
@@ -43,6 +43,7 @@ function _install_package(manager_name, package_name, opt)
         table.insert(managers, "vcpkg")
         table.insert(managers, "brew")
     end
+    table.insert(managers, "clib")
     assert(#managers > 0, "no suitable package manager!")
```

Comment:
Please do not modify this file, We only need to add the package/manager/clib directory.